bestpy/answers.py

```diff
@@ -122,7 +122,29 @@ def os() -> str:
         "physiology",
     ],
     "teacher_name": ["Dave", "Mary", "Frank"],
-    "grade": "A",
+    "console": [
+        "PlayStation 5",
+        "Xbox Series X",
+        "Xbox Series S",
+        "PlayStation 4 Pro",
+        "PlayStation 4 Slim",
+        "PlayStation 4",
+        "Nintendo Switch",
+        "Xbox One X",
+        "Xbox One S",
+        "Xbox One",
+        "Wii U",
+        "Playstation 3 Super Slim",
+        "PlayStation 3 Slim",
+        "PlayStation 3",
+        "Xbox 360",
+        "Wii",
+        "PlayStation 2",
+        "PlayStation",
+        "Xbox",
+        "Gamecube"
+    ],
+    "grade": "A+",
```

Comment:
```suggestion
    "grade": "A",
```
We could work on that in a future PR (or at least commit). Remember to keep them "atomic", so every commit has a specific purpose and does a single thing.